lib/modules/blockchain_process/blockchain.js

```diff
@@ -178,9 +179,12 @@ Blockchain.prototype.run = function() {
       console.error(`Geth error: ${data}`);
     });
     // Geth logs appear in stderr somehow
-    self.child.stderr.on('data', (data) => {
+    self.child.stderr.on('data', async (data) => {
       data = data.toString();
       if (!self.readyCalled && data.indexOf('WebSocket endpoint opened') > -1) {
+        if (self.config.proxy) {
```

Comment:
So we want the proxy running in testnet/mainnet situations? As an example, the proxy might try forward `localhost:90` requests to `ropsten.infura.io:80`? I definitely could be wrong.

Reply:
above: `self.child = child_process.spawn(...)`

The callback in question pertains to: `self.child.on(...)`

I assumed that would only be relevant to a local process (e.g. a geth that embark spawned). I could be wrong about that! Thoughts?